datafusion/sqllogictest/test_files/window.slt

```diff
@@ -1767,27 +1767,25 @@ logical_plan
 02)--Aggregate: groupBy=[[]], aggr=[[count(Int64(1)) AS count(*)]]
 03)----SubqueryAlias: a
 04)------Projection: 
-05)--------Sort: aggregate_test_100.c1 ASC NULLS LAST
-06)----------Aggregate: groupBy=[[aggregate_test_100.c1]], aggr=[[]]
-07)------------Projection: aggregate_test_100.c1
-08)--------------Filter: aggregate_test_100.c13 != Utf8("C2GT5KVyOPZpgKVl110TyZO0NcJ434")
-09)----------------TableScan: aggregate_test_100 projection=[c1, c13], partial_filters=[aggregate_test_100.c13 != Utf8("C2GT5KVyOPZpgKVl110TyZO0NcJ434")]
+05)--------Aggregate: groupBy=[[aggregate_test_100.c1]], aggr=[[]]
+06)----------Projection: aggregate_test_100.c1
+07)------------Filter: aggregate_test_100.c13 != Utf8("C2GT5KVyOPZpgKVl110TyZO0NcJ434")
+08)--------------TableScan: aggregate_test_100 projection=[c1, c13], partial_filters=[aggregate_test_100.c13 != Utf8("C2GT5KVyOPZpgKVl110TyZO0NcJ434")]
 physical_plan
 01)ProjectionExec: expr=[count(*)@0 as global_count]
 02)--AggregateExec: mode=Final, gby=[], aggr=[count(*)]
 03)----CoalescePartitionsExec
 04)------AggregateExec: mode=Partial, gby=[], aggr=[count(*)]
-05)--------RepartitionExec: partitioning=RoundRobinBatch(2), input_partitions=2
```

Comment:
This seems like an unnecessary repartition that shouldn't have been here in the first place. How did our removal of the logical `Sort` result in this being absent in the new physical plan?

We may have discovered a small bug in `EnforceDistribution` (or some other physical rule) that we should fix. Let's investigate.